.github/ISSUE_TEMPLATE/falsepositive.yml

```
@@ -0,0 +1,112 @@
+# https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms
+name: False Positive Report
+description: File a Phishing report
+labels: ["False Positive"]
+assignees:
+  - mitchellkrogza
+  - funilrys
+body:
+  - type: markdown
+    attributes:
+      value: |
+        We understand being listed on a Phishing Database like this can be frustrating and embarrassing for many web site owners.
+
+        The first step is to remain calm.
+        
+        The second step is to rest assured one of our maintainers will address your issue as soon as possible.
+
+        Please make sure you have provided as much information details as possible to help speed up the process.
+
+  - type: textarea
+    id: record
+    attributes:
+      label: Domain/URL/IP(s) you believes NOT to be Phishing
```

Does this label really need to be that "long" ?

---

```suggestion
      label: Domain/URL/IP(s) you believe NOT to be Phishing
```
There seems to be typo.


Also, maybe this could be _Website(s) you believe to be false positives_

---

👍🏾 @spirillen, this one is yours.